srcpkgs/Kooha/template

```diff
@@ -1,22 +1,24 @@
 # Template file for 'Kooha'
 pkgname=Kooha
-version=2.0.1
+version=2.2.3
 revision=1
 build_style=meson
 build_helper="rust"
-hostmakedepends="pkg-config gettext glib-devel cargo"
-makedepends="libglib-devel gtk4-devel libadwaita-devel rust-std"
+hostmakedepends="cargo desktop-file-utils gettext glib glib-devel pkg-config"
```

Comment:
Might need `gtk-update-icon-cache` in `hostmakedepends` for cross

Reply:
That fixed it, thanks!